eus-test.l

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env roseus
 (require :unittest "lib/llib/unittest.l")
 
+(load "auxiliary/eus-multiple-values.l")
```

Comment:
maybe shift this line below, around `;;LOAD AUXILIARY` at #L135?

Reply:
the thing is that that file might be of use not only for the test files, but for `eus-test.l`as well, so I thought leaving it closer to the header would make that clearer. but I could change it, it is not a problem

Reply:
for now `eus-test.l` is not using any multiple-value so it should be fine.

Another possibility is to bring `eus-loop.l` here, so we can use both multiple-values and loop in the rest of the file.